### PR TITLE
Admin manual input for ticket amount

### PR DESCRIPTION
**Pull request by guainumbi**
This PR adds an input field to the admin event show view and organizers event show view. Admins and organizers can enter or edit the amount of distributed tickets for an event manually here, regardless of the application process. The amount will be saved in the database with the according event so it can later show up in the annual events report statistics. This way we can help making the statistics more accurate for the number of distributed tickets via Diversity Tickets, even if the selection process is not being handled by DT.
The feature also gives a quick visual overview of how many tickets have been given out/ applications have been approved so far.

### CODE REVIEW

**Comment by mauditecandela** [comment]
captura de pantalla 2018-08-13 a las 12 34 42

File: app/controllers/admin_events_controller.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 class AdminEventsController < ApplicationController
   before_action :get_event, except: [:index, :annual_events_report]
   before_action :require_admin
+  before_action :get_approved_tickets_count, only: [:show]
```
Comment (mauditecandela) [inline app/controllers/admin_events_controller.rb]: @guainumbi Is this strictly necessary? If yes, I don't understand the use. Could you please explain?

Comment (guainumbi) [reply app/controllers/admin_events_controller.rb]: This is updating the database with the approved_tickets_count of an event (when event is created the value is nil at first, the first view of the events admin_show view updates the db with current count. from then on we add +1 to the count whenever we approve an application). I guess we can also call the method in the show view, but I thought it is cleaner like this. Another solution would be to have the default value of approved_ticket_count as 0...

Comment (mauditecandela) [reply app/controllers/admin_events_controller.rb]: I think we should have a default value in the db better, but I will leave you to your choice. What about the UI comment I left on top of this comment? :) Are you thinking about modifying it?

Comment (guainumbi) [reply app/controllers/admin_events_controller.rb]: changing to integer with default value. yes, i think it makes sense to think about different layout.

**Comment by guainumbi** [comment]
@mauditecandela the view from your screenshot is only an admins view for admin events details. the user will see the following in the event show view: 
bildschirmfoto 2018-08-14 um 11 45 51

**Pull request merged**